app/src/main/java/co/tinode/tindroid/services/FBaseMessagingService.java

```diff
@@ -338,10 +339,14 @@ private void handleCallNotification(@NonNull String webrtc, boolean isMe, @NonNu
                     }
                     break;
                 case "accepted":
+                    CallInProgress call = Cache.getCallInProgress();
+                    if (origSeq > 0 && call != null && call.isConnected() && call.equals(topicName, origSeq)) {
```

Comment:
In this line `call.equals(topicName, origSeq)` returns false because `setCallActive` is not yet called and `mSeq` is 0.